tests/BenchmarkDotNet.Tests/Reports/SummaryTableTests.cs

```diff
@@ -46,19 +46,21 @@ public void PlatformTest()
         }
 
         [Fact]
-        public void NumericColumnIsRightJustified()
```

Comment:
Please add new tests rather than overwriting the old tests.

Reply:
For these two tests, The previous intention was completely outdated and not really working. the justify value was set for columns but it was only used to justify header separator indicator. Also please note that since numeric columns are not justified right anymore (unless user sets newly added justify style to right) and it had caused the tests to fail. I updated the tests for new design.
What do you suggest?

Reply:
I suggest fixing the old tests. You may need to add a separate option for text justification and numeric justification. Header justification may or may not need its own option also, but I'm not strongly opinionated on that.